api.yaml

```
@@ -303,6 +303,8 @@ components:
         bookmarkable:
           type: boolean
           example: true
+        AdControllerPageInfo:
```

Wir haben die Konvention, das Properties klein geschrieben werden, also `adControllerPageInfo`. Beim Schema ist die Schreibweise richtig.
Property: dromedary case (initial lowercase letter)
Schema: Pascal case (initial uppercase letter)
```suggestion
        adControllerPageInfo:
```

---

naming things 🙈